charts/common/templates/_regions.tpl

```diff
@@ -0,0 +1,200 @@
+{{- define "sysdig.regions" -}}
```

Comment:
I can't find anywhere in the IBM documentation where these endpoints are defined:
```
- <region>.sysdig-secure.cloud.ibm.com
- ingest.<region>.sysdig-secure.cloud.ibm.com
- ingest.private.<region>.sysdig-secure.cloud.ibm.com
- private.<region>.sysdig-secure.cloud.ibm.com
```